paragraph/scheduling/instruction_fsm.h

```diff
@@ -85,6 +85,8 @@ class InstructionFsm {
   double GetTimeFinished();
   void SetTimeFinished(double current_time);
 
+  std::string ToCsv(const std::string& delimeter = ",") const;
```

Comment:
Technically, "CSV" stands for comma separated values.

Reply:
Right, that's why default delimiter is comma. Yet excel offers you to pick a delimiter symbol, that's why it is a parameter